landscape/lib/network.py

```diff
@@ -138,20 +139,30 @@ def get_flags(sock, interface):
     return struct.unpack("H", data[16:18])[0]
 
 
-def get_active_device_info(skipped_interfaces=("lo",),
-                           skip_vlan=True, skip_alias=True, extended=False):
+def get_virtual():
+    """
+    Returns a list of virtual interfaces see link:
+    https://unix.stackexchange.com/a/58328
+    """
+    try:
+        interfaces = os.listdir('/sys/devices/virtual/net')
+    except OSError:
+        interfaces = []
+    return interfaces
+
+
+def get_active_device_info(skip_alias=True, extended=False):
     """
     Returns a dictionary containing information on each active network
     interface present on a machine.
     """
     results = []
+    virtual_interfaces = set(get_virtual())
     try:
         sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM,
                              socket.IPPROTO_IP)
         for interface, ifaddresses in get_active_interfaces():
-            if interface in skipped_interfaces:
-                continue
-            if skip_vlan and "." in interface:
+            if interface in virtual_interfaces:
```

Comment:
Have you looked at the various callsites of this function?
It looks like this is used by a networkdevice monitor, which would alter monitoring of those interfaces by landscape-client.
This looks like it's altering a bit more than just sysinfo.